starter-code/css/main.css

```diff
@@ -6,3 +6,130 @@ Green: #00B172
 White: #FFF
 
 */
+
+/* ---- reset ---- */
+
+html {
+  box-sizing: border-box;
+}
+*, *:before, *:after {
+  box-sizing: inherit;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+
+
+
+/*----spotify-nav-----*/
+
+.spotify-nav-ul,
+.spotify-nav-ul li {
+  float: right;
+  display: inline;
+}
+.spotify-nav-ul a {
+  text-decoration: none;
+  padding: 6px;
+}
+.spotify-nav img {
+  padding: 10px;
+  margin-left: 30px;
+}
+
+
+/*----header-spotify----*/
+.header-spotify {
+  background: url("./../images/landing.jpg");
+  background-repeat: no-repeat; 
+  height: 100vh;
+  background-size: cover;
+  background-position: center;
+  text-align: center;
+
+}
+.header-spotify h1 {
+  margin: 0px;
+}
+
+
+
+/*----about-spotify----*/
+.about-spotify {
+  margin: 0 auto;
+  text-align: center;
+  margin-bottom: 400px;
+}
+.about-spotify h2 {
+  text-align: center;
+
+}
+.col {
+  float: left;
+  margin-left: 30px;
+}
+.col-3 {
+  width: 31%;
+}
```

Comment:
Good stuff :+1: In smaller screens you normally do not have those 30px per item in the margin-left.